utils/src/test/java/se/fortnox/reactivewizard/util/ReflectionUtilTest.java

```diff
@@ -299,6 +299,12 @@ public void shouldSupportCglibProxiesWhenLocatingOverriddenMethods() throws Exce
         assertThat(method).isEqualTo(TestResource.class.getMethod("resourceGet", String.class, String.class));
     }
 
+    @Test
+    public void shouldGetRedefinedVersionOfMethod() throws NoSuchMethodException {
```

Comment:
This just test that you will get the same method. Testing a new version of the exact same class is beyond my skills. You would have to use CGLib or ASM to pull that off, and we even have code inside `getUserDefinedClass` which circumvents that, so I have no clue how to write a test that shows that hot reload is really working.

Tested it manually though.